docs/openapi.yaml

```diff
@@ -46,6 +46,24 @@ components:
       description: Evaluate the contract at up to a specific sort key. Only applicable if blockHeight is not provided.
       schema:
         type: string
+    page:
+      name: page
+      in: query
+      required: false
+      description: The page of results to return. 1 indexed. Minimum of 1, maximum of MAX_SAFE_INTEGER.
+      schema:
+        type: number
+        maximum: MAX_SAFE_INTEGER
+        minimum: 1
```

Comment:
Won't page 1 be ever changing? On the next request for page 2, some of page 1 may be on it. Should we use some sort of last interaction anchor?